### PR TITLE
New Colouring Core Preparations

### PR DESCRIPTION
**Pull request by mdsimpson42**
This is the first step towards creating the new 'colouring core' repo. A config file is used to define the city name, GitHub URLs and various other properties, which are then used throughout the site, making them easier to customise. More changes are incoming; I just wanted to get this first step working before adding any other changes.

Once this pull request has been merged, we will rename the current "colouring-london" repository to "colouring-core" and create a new "colouring-london" repository as a fork from that, which will put London in line with all of the other projects, with a single common base project.

### CODE REVIEW

**Comment by matkoniecz** [comment]
> Once this pull request has been merged, we will rename the current "colouring-london" repository to "colouring-core" and create a new "colouring-london" repository as a fork from that, which will put London in line with all of the other projects, with a single common base project.

Note that we should then update urls in our remotes

File: app/src/frontend/pages/privacy-policy.tsx [hunk 1]
```diff
@@ -1,49 +1,52 @@
...
 
             <h2 className='h2'>What is the legal basis for processing your data?</h2>
             <p>
-                Data protection laws require us to meet certain conditions before we are allowed to use your data in the manner described in this notice, including having a ‘legal basis’ for the processing. Colouring London, as a research project, is processing your personal data in pursuance of its legitimate interests.
+                Data protection laws require us to meet certain conditions before we are allowed to use your data in the manner described in this notice, including having a ‘legal basis’ for the processing. Colouring {config.cityName}, as a research project, is processing your personal data in pursuance of its legitimate interests.
             </p>
 
             <h2 className='h2'>How do we store your data?</h2>
             <p>
-                Colouring London stores your data at The Alan Turing Institute in London behind the organisation’s firewall in a secure database using industry standard practices.
```
Comment (matkoniecz) [inline app/src/frontend/pages/privacy-policy.tsx:41]: > at The Alan Turing Institute in London

this does not apply to all CCRP installs, right? And also should be configurable

Comment (mdsimpson42) [reply app/src/frontend/pages/privacy-policy.tsx:41]: Yes, good point. I can change this now, and add the commit to the pull request. Any other changes like this we can add as issues and fix after the merge?

**Comment by mdsimpson42** [comment]
> > Once this pull request has been merged, we will rename the current "colouring-london" repository to "colouring-core" and create a new "colouring-london" repository as a fork from that, which will put London in line with all of the other projects, with a single common base project.
> 
> Note that we should then update urls in our remotes

Yeah, there will be some tidying up and additional tweaks to be made after the merge before it's fully ready.

**Pull request merged**